src/scss/components/_product-variant.scss

```diff
@@ -24,13 +25,41 @@
 
 		&:checked ~ .custom-control-label {
 			border-color: $primary !important;
-			.product-swatch {
+			.product-swatch,
+			.product-variant__note {
 				display: block;
 			}
 		}
 	}
 
+	&--subscription {
+		.subscription-title {
```

Comment:
please use BEM convention, it must be subscription__title

Reply:
Ops let me update

Reply:
Updated